### DISCUSSION TITLE
Restore qemu-img?

### DISCUSSION THREAD

**Discussion by fdcastel** [pauby/ChocoPackages#193] [Ideas]
Could we restore the `qemu-img` chocolatey package? 

It is very useful when you just want to handle images but have no need to install the full `qemu` package.

I'm willing to keep it updated with the latest versions from https://qemu.weilnetz.de/w64.

I already did the initial work and put the latest (stable) binaries [here](https://github.com/fdcastel/qemu-img-binaries), for reference. It's the minimal set of files needed (extracted from official installer).

**Comment by pauby** [comment]
As it sounds like you've done most of the work already required here, I'm happy to turn the package over to you to maintain it. Can you send an email using the 'Contact Maintainers' link on the package page and let me know your Chocolatey username?

One thing I would say is that you will not be able to use your repository as the source for the binaries in the package. They need to be taken from the official source.

**Comment by fdcastel** [comment]
> One thing I would say is that you will not be able to use your repository as the source for the binaries in the package. They need to be taken from the official source.

I understand. I believe the installer was built with Wix. I will look for some way of extract the files. 

Should we close this issue? Or we can keep it open for me to report the progress here.

**Reply by pauby** [reply]
As you're picking the package up, and I won't be involved in maintenance, it makes sense to keep the progress over on your repository.

**Reply by fdcastel** [reply]
Sure! I understand. 👍 

One last question: where could I find the **original (non-deprecated) sources** for this package (prior to Jan/2023)?

A quick search in your repository bring me only [this commit](https://github.com/pauby/ChocoPackages/commit/725c48027de2b9886ff3b86e0cb1099c0ca58070).

**Reply by pauby** [reply]
I didn't maintain the package before then so what you're pointing to is the source I changed it into to deprecate it.

The previous package version is [here](https://community.chocolatey.org/packages/qemu-img/2.3.0) and the repository for the source is [here](https://bitbucket.org/fk/misc-chocolatey) which no longer exists. What I'd do is download the package from [the previous version](https://community.chocolatey.org/packages/qemu-img/2.3.0) and use that as the source.
